packages/app/src/cli/services/dev/fetch.test.ts

```diff
@@ -200,6 +273,9 @@ describe('NoOrgError', () => {
       │  No Organization found                                                       │
       │                                                                              │
       │  Next steps                                                                  │
+      │    • Your current active session is asscociated with the unkonwn account.    │
```

Comment:
this is weird: "... with the unknown account" 
It should be `... is associated with an unknown account`